### ISSUE TITLE
Continuous integration

### ISSUE THREAD

**Issue by hymerman** [unittest-cpp/unittest-cpp#68]
I think we should set up CI using Travis (for linux) and AppVeyor (for Windows). I'm quite happy to do the work (I've just done it for another project); I just thought I'd create an issue before I start to make sure I'm not treading on anyone's toes!

**Comment by pjohnmeyer** [comment]
I'm not opposed; I was thinking about setting up a Jenkins server, and letting anybody attach a slave with their particular tool chain to it. This would allow for a more diverse set of build systems to run the tests. But, anything would be better than what we have right now. If you want to invest the time I say go for it.

**Comment by hymerman** [comment]
Wow, sorry I took so long with this - it got put on my todo list and then immediately buried by a hundred other things. I've set it up now - have a look at the linked PR. Should be good to go after you've set up accounts on both services - should only take 5 mins.

A Jenkins server would be fantastic long-term, but Travis and AppVeyor are just *so* damn convenient! And reliable too - Jenkins has always been a 99% automatic solution for me.
